ReelTalk/screens/RecommendScreen.js

```diff
@@ -4,35 +4,28 @@ var React = require('react-native');
 var {
   AppRegistry,
   StyleSheet,
-  NavigatorIOS,
   Text,
   View,
 } = React;
 
-var RecommendHome = require('../containers/RecommendHome');
+import RecommendHome from '../containers/RecommendHome';
 
 var RecommendScreen = React.createClass({
   render: function() {
     return (
-      <NavigatorIOS
```

Comment:
Ya i did like 2-3 hours of reading and coding while watching 30 Rock. So I was a bit distracted but I definitely think I'm closer to the right solution,